File: C++/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Exercises/Solutions/P01_Cinema.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {

	double premiere = 12;
	double normal = 7.5;
	double discount = 5;

	string type, strR, strC;
	getline(cin, type);
	getline(cin, strR);
	int rows = stoi(strR);
	getline(cin, strC);
	int cols = stoi(strC);

	double income = 0.0;
	if (type == "Premiere") {
		income = premiere * rows * cols;
	} else if (type == "Normal") {
		income = normal * rows * cols;
	} else if (type == "Discount") {
		income = discount * rows * cols;
	}

	cout.setf(ios::fixed);
	cout.precision(2);

	cout << income << " leva" << endl;

	return 0;
}

File: Python/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Exercises/Solutions/P01_Cinema.py
type = input()
rows = int(input())
columns = int(input())

price = 0

Premiere = 12
Normal = 7.50
Discount = 5

if type == "Premiere":
    price = rows * columns * Premiere
elif type == 'Normal':
    price = rows * columns * Normal
elif type == 'Discount':
    price = rows * columns * Discount


print(f'{price:.2f}' ' leva')